lmdeploy/cli/cli.py

```diff
@@ -31,15 +31,15 @@ def add_parser_convert():
             description=CLI.convert.__doc__,
             help=CLI.convert.__doc__)
         # define arguments
-        parser.add_argument(
```

Comment:
this is a BC for `lmdeploy convert` command, should deprecate at first and then remove later?  maybe need to strongly mentioned it in release note later.

Reply:
Not BC any more